File: PianoRiseDotnet/PianoRiseApp/Models/AlumnoAula.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PianoRiseManagement.Models
{
    [Table("alumno_aula")]
    public class AlumnoAula
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("alumno_id"), ForeignKey("Alumno")]
        public int AlumnoId { get; set; }

        [Column("aula_id"), ForeignKey("Aula")]
        public int AulaId { get; set; }

        [Column("inscrito_en")]
        public DateTime InscritoEn { get; set; }

        public Usuario Alumno { get; set; } = null!;
        public Aula Aula { get; set; } = null!;
    }
}

File: PianoRisePython/app/models/alumno_aula.py
from datetime import datetime
from sqlalchemy import Column, Integer, DateTime, ForeignKey
from sqlalchemy.orm import relationship
from app import db

class AlumnoAula(db.Model):
    __tablename__ = "alumno_aula"

    id           = Column("id", Integer, primary_key=True)
    alumno_id    = Column("alumno_id", Integer, ForeignKey("usuario.id"), nullable=False)
    aula_id      = Column("aula_id",   Integer, ForeignKey("aula.id"),    nullable=False)
    inscrito_en  = Column("inscrito_en", DateTime, default=datetime.utcnow, nullable=False)

    alumno = relationship("Usuario",    back_populates="alumno_aulas")
    aula   = relationship("Aula",       back_populates="alumno_aulas")
